src/redux/actions/concepts/dictionaryConcepts.js

```diff
@@ -144,6 +144,24 @@ export const fetchDictionaryConcepts = (
   dispatch(isFetching(false));
 };
 
+export const fetchConceptsByName = query => async (dispatch) => {
+  dispatch(isFetching(true));
+  const conceptType = localStorage.getItem('type');
+  const userTypeName = localStorage.getItem('typeName');
+  const dictionaryId = localStorage.getItem('dictionaryId');
+  const limit = 0;
```

Comment:
Have you looked at the naming convention for constants?

Reply:
@dkayiwa I have changed the constant naming to use an accepted convention.